src/devtools/client/debugger/src/components/shared/InspectorContextReduxAdapter.tsx

```diff
@@ -80,7 +80,7 @@ export default function InspectorContextReduxAdapter({ children }: { children: R
               objectId = object.preview?.node?.parentNode;
             }
 
-            dispatch(openNodeInInspector(valueFront));
+            // dispatch(openNodeInInspector(valueFront));
```

Comment:
Did you mean to delete this or did you accidentally commit the commented out version?

Reply:
Intentional. This function isn't actually hooked up at all right now anyway, and this was the _only_ function that calls `openNodeInInspector`.  Given that, and that my immediate goal is to rip out the `NodeFront` stuff, I just commented this out to let me keep going with cleanup.

Reply:
Restored this since it led to a linter "don't need `[dispatch]` deps" warning